py3status/modules/google_calendar.py

```diff
@@ -12,12 +13,11 @@
     blacklist_events: Event names in this list will not be shown in the module
         (case insensitive).
         (default [])
-    button_open: Opens the URL for the clicked on event in the default
-        web browser.
+    button_open: Opens the event URL in the default web browser.
         (default 3)
     button_refresh: Refreshes the module and updates the list of events.
         (default 2)
-    button_toggle: The button used to toggle what data is displayed for each event.
+    button_toggle: Toggles a boolean to hide/show the data for each events.
```

Comment:
__event__ or else different wording

Reply:
It's finished. That's what I was changing just now... Ha.